wagtail/images/templates/wagtailimages/bulk_actions/confirm_bulk_delete.html

```diff
@@ -0,0 +1,50 @@
+{% extends 'wagtailadmin/bulk_actions/confirmation/base.html' %}
+{% load wagtailadmin_tags i18n %}
+{% block titletag %}{% blocktrans count counter=images|length %}Delete 1 image{% plural %}Delete {{ counter }} images{% endblocktrans %}{% endblock %}
+
+{% block header %}
+    {% trans "Delete images" as del_str %}
+    {% include "wagtailadmin/shared/header.html" with title=del_str icon="doc-full-inverse" %}
+{% endblock header %}
+
+{% block objects_with_access %}
+        {% if images %}
+        <p>
+            {% blocktrans count counter=images|length %}
+            Are you sure you want to delete this image?
+            {% plural %}
+            Are you sure you want to delete these images?
+            {% endblocktrans %}
+        </p>
+        <ul>
+        {% usage_count_enabled as uc_enabled %}
+        {% for image in images %}
+            <li>
+                <a href="{% url 'wagtailimages:edit' image.image.id %}"  target="_blank" rel="noopener noreferrer">{{image.image.title}}</a>
+                {% if uc_enabled %}
+                <p>
+                    <a href="{{ image.image.usage_url }}">{% blocktrans count usage_count=image.image.get_usage.count %}Used {{ usage_count }} time{% plural %}Used {{ usage_count }} times{% endblocktrans %}</a>
+                </p>
+                {% endif %}
```

Comment:
```suggestion
                <a href="{% url 'wagtailimages:edit' image.image.id %}"  target="_blank" rel="noopener noreferrer">{{image.image.title}}</a> 
                {% if uc_enabled %} (
                    <a href="{{ image.image.usage_url }}">{% blocktrans count usage_count=image.image.get_usage.count %}Used {{ usage_count }} time{% plural %}Used {{ usage_count }} times{% endblocktrans %}</a>)
                {% endif %}
```
would make the list tighter:
![bulk-usage-tight](https://user-images.githubusercontent.com/31622/128634868-22e5831d-9272-43cf-a334-fd7fabf08a9d.png)

as opposed to:
![bulk-usage-lcurrent png](https://user-images.githubusercontent.com/31622/128634871-ffe415e9-7ddb-40b0-8ae5-2d1edf19bac9.png)


What do you think?

Reply:
Yes, the top one looks more compact, and the weight of the usage string is also not lost